man/pcrelate.Rd

```diff
@@ -68,7 +68,7 @@ correctK0(kinBtwn)
   \item{sample.block.size}{The number of individuals to read-in/analyze at once; the default value is 5000. See 'Details' for more information.}
   \item{maf.thresh}{Minor allele frequency threshold; if an individual's estimated individual-specific minor allele frequency at a SNP is less than this value, that indivdiual will either have that SNP excluded from the analysis or have their estimated indivdiual-specific minor allele frequency truncated to this value, depending on \code{maf.bound.method}. The default value is 0.01.}
   \item{maf.bound.method}{How individual-specific minor allele frequency estimates less that \code{maf.thresh} are handled. When set to `filter` (default), SNPs for which an individual's estimated individual-specific minor allele frequency are below \code{maf.thresh} are excluded from the analysis for that individual. When set to `truncate`, estimated individual-specific minor allele frequncies below \code{maf.thresh} have their value set to \code{maf.thresh}.}
- \item{small.samp.correct}{Logical indicator of whether to implement a small sample correction.}
+ \item{small.samp.correct}{Logical indicator of whether to implement a small sample correction. The default is \code{TRUE}, but must be set to \code{FALSE} if \code{sample.block.size} is less than the number of samples or if \code{scale} is \code{'none'}.}
```

Comment:
@mconomos should we add any additional instructions for when it's advised to set it to `FALSE`?

Reply:
It will automatically switch to false when the number of sample blocks is greater than 1 (and print a message). We could say something like "it's computationally demanding on large samples, so we don't recommend it if your sample size is > 10,000(?)"

I think we _did_ use it for OLGA, which was like 12,000, but probably less necessary.

Reply:
(sorry i missed this question before you merged)